charts/stan/values.yaml

```diff
@@ -161,13 +165,14 @@ store:
 #######################################
 exporter:
   enabled: true
-  resources: {}
-  #  limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  #  requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
+    limits:
+     cpu: 100m
+     memory: 100Mi
+    requests:
+     cpu: 100m
```

Comment:
Requests: you can change it to CPU 50m, memory to 80Mi

Reply:
https://github.com/astronomer/astronomer/blob/master/values.yaml#L401-L413